hystrix-core/src/test/java/com/netflix/hystrix/HystrixCommandTest.java

```diff
@@ -2766,6 +2766,36 @@ public void testDoNotInterruptToObservableOnTimeoutIfPropertySaysNotTo() throws
     }
 
     @Test
```

Comment:
Can you also add a test when cancel(false) is called?  In this case, I would expect the Hystrix thread to keep sleeping, and the latch timeout to get hit in the unit test